BlueprintUICommonControls/Sources/AttributedLabel.swift

```diff
@@ -556,7 +556,11 @@ extension AttributedLabel {
             // Insert the word "link" after each link in the label. This mirrors the VoiceOver behavior when encountering a `.link` attribute.
 
             guard let localizedLinkString = linkAccessibilityLabel,
-                  !links.isEmpty else { return string }
+                  !links.isEmpty
+            else {
+                // We need to replace all newlines with " "
+                return string.components(separatedBy: .newlines).filter { !$0.isEmpty }.joined(separator: " ")
```

Comment:
This makes me question where this was happening before...

Reply:
Probably the default value coming from UIKit?

Reply:
Yup, that's what i think too

Reply:
yea, i tested it and it seems to be happening inside UILabel and only occurs when you don't provide an explicit label.